pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyLanguageSql.java

```diff
@@ -41,4 +46,17 @@ int updateDefaultLanguage(@Bind("studyId") long studyId,
             + " where umbrella_study_id = :studyId and is_default = true")
     List<Long> selectDefaultLanguageCodeId(long studyId);
 
+    @SqlQuery("select language_code.iso_language_code as 'languageCode', "
+            + "study_language.name as 'displayName', "
+            + "study_language.is_default as 'isDefault', "
+            + "study_language.umbrella_study_id as 'studyId',"
+            + "study_language.language_code_id as 'languageId' "
+            + "FROM language_code, study_language "
```

Comment:
Can we just `join` syntax here? E.g. `from study_language as sl join language_code as lc on ...`.